developer_docs/foreman_dev_setup.asciidoc

```diff
@@ -0,0 +1,115 @@
+[[Containers]]
+= Install From Source
+:toc: right
+:toclevels: 5
+
+Installing the latest development code:
+Foreman uses Bundler to install dependencies and get up and running. This is the preferred way to get Foreman if you want to benefit from the latest improvements. By using the git repository you can also upgrade more easily. You will need to have Bundler installed manually for now (check your distribution repositories, or install it via rubygems).
+
+[[prerequisites]]
+== Prerequisites
+Foreman will run with the following requirements (aside from rubygem dependencies):
+
+* Ruby v2.7.x
+* NodeJS v12 or v14
+* NPM v6
+* PostgreSQL v13
+
+It’s recommended to install https://github.com/rbenv/rbenv[rbenv] to manage ruby, and https://github.com/nvm-sh/nvm[nvm] to manage node and npm on your system.
+Before installing PostgreSQL, check first that you don't already have it installed (by running `rpm -q postgresql`).
+
+You will want to make sure that you have postgresql-devel installed so the database gems can install properly.
+Also, you would also need gcc, ruby-devel, libxml-devel, libxslt-devel and libvirt-devel packages.
+
+For RHEL7 or clones, you can issue the following commands to install all required packages:
```

Comment:
```suggestion
For Fedora,  CentOS or RHEL you can issue the following commands to install all required packages:
```

Reply:
Could be as a follow-up PR, currently I don't have a Debian setup.
Do you know any developer who uses Debian?

Reply:
Would be nice to have it but I would be also happy with leaving that part for the contribution from community.
The only difference would be installing different packages for ruby+rbenv and that can be easily solved by one search in google